benchmarks/dynamo/torchao_backend.py

```diff
@@ -14,7 +14,7 @@ def setup_baseline():
 
 def torchao_optimize_ctx(quantization: str):
     import torchao
-    from torchao.quantization import (
+    from torchao.quantization.quant_api import (
```

Comment:
for nightly please use the 2.4+ APIs, e.g.: https://github.com/pytorch/ao/tree/main/torchao/quantization#a16w8-weightonly-quantization